architectures.py

```diff
@@ -101,7 +98,7 @@ def __init__(self, config):
             nn.ReLU(),
         )
 
-        head_inputs = ROWS * COLS * config.kernels + config.o_side_neurons + 2 * ((2 + PREVIEWS) * len(MINOS)) + 2 * 4 + 2
```

Comment:
Try to avoid magic numbers if you can
https://en.wikipedia.org/wiki/Magic_number_(programming)